eos/form-factors/mesonic_TEST.cc

```diff
@@ -759,3 +760,161 @@ class BToKstarBSZ2015FormFactorsTest :
+        p["B->rho::alpha^V_2@BSZ2015"  ] = +1.80;
+        p["B->rho::alpha^T1_0@BSZ2015" ] = +0.27;
+        p["B->rho::alpha^T1_1@BSZ2015" ] = -0.74;
+        p["B->rho::alpha^T1_2@BSZ2015" ] = +1.45;
+        p["B->rho::alpha^T2_0@BSZ2015" ] = +0.27;
+        p["B->rho::alpha^T2_1@BSZ2015" ] = +0.47;
+        p["B->rho::alpha^T2_2@BSZ2015" ] = +0.58;
+        p["B->rho::alpha^T23_0@BSZ2015"] = +0.75;
+        p["B->rho::alpha^T23_1@BSZ2015"] = +1.90;
+        p["B->rho::alpha^T23_2@BSZ2015"] = +2.93;
+
+        TEST_CHECK_NEARLY_EQUAL(0.36186,  ff->a_0(0.1), eps);
+        TEST_CHECK_NEARLY_EQUAL(0.402772, ff->a_0(2.1), eps);
+        TEST_CHECK_NEARLY_EQUAL(0.4521,   ff->a_0(4.1), eps);
+        TEST_CHECK_NEARLY_EQUAL(0.512422, ff->a_0(6.1), eps);
+
+        TEST_CHECK_NEARLY_EQUAL(0.260532, ff->a_1(0.1), eps);
+        TEST_CHECK_NEARLY_EQUAL(0.271749, ff->a_1(2.1), eps);
+        TEST_CHECK_NEARLY_EQUAL(0.284225, ff->a_1(4.1), eps);
+        TEST_CHECK_NEARLY_EQUAL(0.29821,  ff->a_1(6.1), eps);
+
+        TEST_CHECK_NEARLY_EQUAL(0.226525, ff->a_2(0.1), eps);
+        TEST_CHECK_NEARLY_EQUAL(0.247972, ff->a_2(2.1), eps);
+        TEST_CHECK_NEARLY_EQUAL(0.272771, ff->a_2(4.1), eps);
+        TEST_CHECK_NEARLY_EQUAL(0.301645, ff->a_2(6.1), eps);
+
+        TEST_CHECK_NEARLY_EQUAL(0.331752, ff->v(0.1), eps);
+        TEST_CHECK_NEARLY_EQUAL(0.370391, ff->v(2.1), eps);
+        TEST_CHECK_NEARLY_EQUAL(0.417199, ff->v(4.1), eps);
+        TEST_CHECK_NEARLY_EQUAL(0.474694, ff->v(6.1), eps);
+
+        TEST_CHECK_NEARLY_EQUAL(0.271458, ff->t_1(0.1), eps);
+        TEST_CHECK_NEARLY_EQUAL(0.303616, ff->t_1(2.1), eps);
+        TEST_CHECK_NEARLY_EQUAL(0.342573, ff->t_1(4.1), eps);
+        TEST_CHECK_NEARLY_EQUAL(0.390422, ff->t_1(6.1), eps);
+
+        TEST_CHECK_NEARLY_EQUAL(0.270508, ff->t_2(0.1), eps);
+        TEST_CHECK_NEARLY_EQUAL(0.28128,  ff->t_2(2.1), eps);
+        TEST_CHECK_NEARLY_EQUAL(0.29338,  ff->t_2(4.1), eps);
+        TEST_CHECK_NEARLY_EQUAL(0.307107, ff->t_2(6.1), eps);
+
+        TEST_CHECK_NEARLY_EQUAL(0.179298, ff->t_3(0.1), eps);
+        TEST_CHECK_NEARLY_EQUAL(0.196067, ff->t_3(2.1), eps);
+        TEST_CHECK_NEARLY_EQUAL(0.215543, ff->t_3(4.1), eps);
+        TEST_CHECK_NEARLY_EQUAL(0.238325, ff->t_3(6.1), eps);
+    }
+} b_to_rho_bsz2015_form_factors_test;
+
+class BToDstarBSZ2015FormFactorsTest :
+public TestCase
+{
+public:
+    BToDstarBSZ2015FormFactorsTest() :
+    TestCase("b_to_Dstar_bsz2015_form_factors_test")
+    {
+    }
+
+    virtual void run() const
+    {
+        static const double eps = 5.1e-3;
+
+        Parameters p = Parameters::Defaults();
+        std::shared_ptr<FormFactors<PToV>> ff = FormFactorFactory<PToV>::create("B->D^*@BSZ2015", p);
+        TEST_CHECK(ff.get() != nullptr);
+
+        /*
+         * for the B->D^* case no SSE parameters exist yet, so for the moment we set them to zero and once the GKvD2018 calculation is fitted one can then simply update this test case accordingly.
```

Comment:
@akokulu7 that's a good idea, thanks! We can then update the test case later on!